src/util/autograder.interface.ts

```diff
@@ -0,0 +1,39 @@
+export interface AutograderData {
+  //What an autograder returns
+  score: number; // optional, but required if not on each test case below. Overrides total of tests if specified.
+  execution_time: number; // optional, seconds
+  output: string; // optional
+  visibility: Visibility; // Optional visibility setting
+  stdout_visibility: Visibility; // Optional stdout visibility setting
+  extra_data: any; // Optional extra data to be stored
```

Comment:
Is it possible to make it a union of types? Or are there an infinite number of structures that this data can take

Reply:
There are infinite number of structures you can come up with because of the optionals